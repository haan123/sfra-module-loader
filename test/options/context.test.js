/* eslint-disable
  prefer-destructuring,
*/
import webpack from '../helpers/compiler';

describe('Options', () => {
  describe('context', () => {
    test('{String}', async () => {
      const config = {
        loader: {
          test: /(js)/,
          options: {
            context: `${__dirname}`
          }
        }
      };

      const stats = await webpack('fixture.js', config);
      const [module] = stats.toJson().modules;
      const { source } = module;

      expect(source).toMatchSnapshot();
    });
  });
});
