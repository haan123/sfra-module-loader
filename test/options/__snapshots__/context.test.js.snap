// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options context {String} 1`] = `
"import core from '*/core';
import component from '*/component';

export default () => {
  core();
  component();
};
"
`;
